src/jit/emit.cpp

```diff
@@ -1219,8 +1219,9 @@ size_t emitter::emitGenEpilogLst(size_t (*fp)(void*, unsigned), void* cp)
     {
         assert(el->elIG->igFlags & IGF_EPILOG);
 
-        UNATIVE_OFFSET ofs =
-            el->elIG->igOffs; // The epilog starts at the beginning of the IG, so the IG offset is correct
+        // The epilog starts epilog_size bytes before the end of the IG. The encoder already has
+        // the epilog size, so hand it the end offset of the IG.
```

Comment:
This creates an implicit requirement that an epilog completely lives within a single IG. This is undesirable, and is not required today. (It is required of function prologs; I worked hard to remove the requirement for all other prolog/epilogs).